citadel/install_windows_src.md

```diff
@@ -0,0 +1,136 @@
+
+1. Install dependencies
+
+        conda install cmake git vcstool curl pkg-config ^
+        colcon-common-extensions eigen freeimage gts ^
+        glib dlfcn-win32 ffmpeg ruby tinyxml2 tinyxml ^
+        protobuf urdfdom zeromq cppzmq ogre jsoncpp ^
+        libzip qt --channel conda-forge
+
+1. Navigate to where you would like to build the library, and then clone the repositories.
+
+        curl -O https://raw.githubusercontent.com/ignition-tooling/gazebodistro/master/collection-citadel.yaml
+        mkdir src
+        vcs import src < collection-citadel.yaml
+
+1. Install `ign-cmake` from `conda-forge` and disable the cloned `ign-cmake` from being built.
+
+        conda install libignition-cmake2 --channel conda-forge
+        touch src\ign-cmake\COLCON_IGNORE
+
+    This is due to a linking error currently existing within the `ign-cmake` source. See the comments [here](https://github.com/ignitionrobotics/docs/issues/96#issuecomment-742096017).
+
+**Note**
+
+You can view all available versions of a specific package with:
+```bash
+conda search libignition-<package_name>* --channel conda-forge
+```
+and view their dependencies with
+```bash
+conda search libignition-<package_name>* --channel conda-forge --info
+```
+and install a specific minor version with
+```bash
+conda install libignition-<package_name>=<major>.<minor>.<patch> --channel conda-forge
+```
+where `<major>` is the major release number, `<minor>` is the minor release number, and `<patch` is the patch release number.
+
+## Building the Ignition Libraries
+
+Once the compiler and all the sources are in place it is time to compile them.
+Start the procedure by changing into the workspace and listing the packages
+recognized by `colcon`:
+
+```bash
+colcon graph
+```
+
+`colcon graph` should list the Ignition libraries with an
+[interdependency diagram](https://colcon.readthedocs.io/en/released/reference/verb/graph.html#example-output).
+If that is the case, then you are ready
+to build the whole set of libraries:
+
+```bash
+colcon build --cmake-args -DBUILD_TESTING=OFF --merge-install --packages-up-to ignition-gazebo3
+```
+Tests are turned off as they are not currently supported on Windows.
+
+To build a specific package with all its dependent packages:
+
+```bash
+colcon build --merge-install --packages-up-to PACKAGE_NAME
+```
+
+To build a single package:
+
+```bash
+colcon build --packages-select PACKAGE_NAME
+```
+
+Visit [colcon documentation](https://colcon.readthedocs.io/en/released/#) to view more `colcon` build and test options.
+
+If there are no errors, all the binaries should be ready to use.
+
+## Using the workspace
+
+The workspace needs to be sourced every time a new terminal is used.
+
+Run the following command to source the workspace:
+
+```bash
+call install\setup.bat
+```
+
+This is the end of the source install instructions; head back to the [Getting started](getting_started.html)
```

Comment:
This ref link needs testing. Actually, none of the currently deployed ones on the website work:
This one links to `[Getting started](getting_started.html)`
https://ignitionrobotics.org/docs/citadel/install_ubuntu_src
This one links to `[Getting started](/docs/all/get_started)`
https://ignitionrobotics.org/docs/citadel/install_osx

Both go to 404 not found.

Reply:
Good catch, I think the correct link is `/docs/all/getstarted` seen here https://ignitionrobotics.org/docs/all/getstarted , I'll look at the other tutorials and see if I can sneak in some link updates

Reply:
Hmm, after taking a closer look, I noticed there possibly being a name aliasing done here: https://github.com/ignitionrobotics/docs/blob/master/index.yaml#L18-L20 , I guess it's a question of should the pages be referenced by their name or file?  I'm assuming it's name if the one's referencing the file are resulting in a 404, do you happen to know anything about this @chapulina ?

Reply:
The current link looks ok to me, it follows the pattern from other tutorials. You're right that the link uses the alias: https://ignitionrobotics.org/docs/all/getstarted